src/network/core/udp.h

```diff
@@ -187,10 +185,17 @@ class NetworkUDPSocketHandler : public NetworkSocketHandler {
 	NetworkUDPSocketHandler(NetworkAddressList *bind = nullptr);
 
 	/** On destructing of this class, the socket needs to be closed */
-	virtual ~NetworkUDPSocketHandler() { this->Close(); }
+	virtual ~NetworkUDPSocketHandler() {
+		/* Virtual functions get called statically in destructors, so make it explicit to remove any confusion. */
+		this->NetworkUDPSocketHandler::CloseSocket();
```

Comment:
Since `CloseSocket` is not virtual anymore, the explicit class is not needed. Likewise for the comment.